function( error ) {
	logger.error( "Error during retry: " + error.stack );
}